source/extensions/filters/http/cache/cacheability_utils.cc

```diff
@@ -0,0 +1,55 @@
+#include "extensions/filters/http/cache/cacheability_utils.h"
+
+#include "common/common/utility.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+
+namespace {
+// As defined by:
+// https://tools.ietf.org/html/rfc7231#section-6.1,
+// https://tools.ietf.org/html/rfc7538#section-3,
+// https://tools.ietf.org/html/rfc7725#section-3
+// TODO(yosrym93): the list of cacheable status codes should be configurable
+const static absl::flat_hash_set<absl::string_view> CacheableStatusCodes = {
```

Comment:
non-POD static variables aren't allowed. In this case, I suggest replacing the use of CacheableStatusCodes with parsing the number and using a switch statement.

Reply:
I can create a static function `isCacheableStatusCode` and define the static `absl::flas_hash_set` inside it, right?

Reply:
yes, inside a function, you can have:
static absl::flash_hash_set& codes = *new absl::flas_hash_set(...